database/seeds/DemoActivityIdsAssociationForActivityItemsSeeder.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class DemoActivityIdsAssociationForActivityItems extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        return DB::transaction(function () {
```

Comment:
Transaction is used only for multiple insertion/update in different tables,
best to move select out of transaction block.